sui-execution/src/lib.rs

```diff
@@ -43,10 +60,31 @@ pub fn verifier<'m>(
     is_metered: bool,
     metrics: &'m Arc<BytecodeVerifierMetrics>,
 ) -> Box<dyn Verifier + 'm> {
-    let version = protocol_config.execution_version_as_option().unwrap_or(0);
+    let version = protocol_config
+        .execution_version_as_option()
+        .unwrap_or(MIN_EXECUTOR_VERSION);
     match version {
         0 => Box::new(v0::Verifier::new(protocol_config, is_metered, metrics)),
-        1 => Box::new(latest::Verifier::new(protocol_config, is_metered, metrics)),
+        LATEST_EXECUTOR_VERSION => {
+            Box::new(latest::Verifier::new(protocol_config, is_metered, metrics))
+        }
         v => panic!("Unsupported execution version {v}"),
     }
 }
+
+/// Sometimes we want to invoke specific versions of the executor regardless of the protocol version
```

Comment:
Could you share a bit more of when we want to change the executor version but not the rest of the protocol config?  Generally the execution version could be coupled with other aspects of the protocol, and I would expect it to be overridden by passing in a protocol config for a different version, or by setting a protocol config override, but maybe I'm missing a use case here.

If we can re-use the test seams that `ProtocolConfig` already provides, that would be preferable.

Reply:
You are correct that we can use protocol config overrides.
I was not aware of that feature.

Reply:
A concrete example is: in many cases people make code changes on latest but want to see the impact of their changes on a historical transaction in replay.